build.gradle

```diff
@@ -149,3 +152,8 @@ subprojects {
     }
 }
 
+apply plugin: 'io.codearte.nexus-staging'
+
+nexusStaging {
+    packageGroup = 'com.pushtorefresh'
```

Comment:
Yeah, unfortunately it'll try to close and release all `com.pushtorefresh` projects that are in staging mode on Sonatype, luckily we don't do releases that often so clash is very unlikely.